checker.js

```diff
@@ -60,14 +58,7 @@ var metadata = {
 
 // Path Strings used
 var tempDirectoryForDiffImages = path.join(os.tmpdir(), 'erc-checker', 'diffImages');
-
-// if tmp directory for erc-checker does not exist already, create it
-try {
-	fs.mkdirSync(path.join(os.tmpdir(), 'erc-checker'));
-}catch (e) {
-
-
-}
+var checkIDglobal = '';
```

Comment:
What is this variable for? "global" IDs and variables are too often a bad idea...

Reply:
This variable name was introduced as a temporary workaround and was not meant to be pushed. It will be gone with the next commit.
There were problems handling the `checkID` parameter of the function when running multiple checks simultaneously. The function I introduced to delete temp files was falsely called with the new  `checkID` of the next call respectively, thus deleting temp data still used by other processes.
This is actually not yet stable and will be fixed with the next commit.